src/test/java/bouncybob/util/ParserTest.java

```diff
@@ -0,0 +1,51 @@
+package bouncybob.util;
+
+import bouncybob.BouncyBob;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+public class ParserTest {
+
+    @Test
+    public void testGetTaskType() {
```

Comment:
Do refer to the Java coding standard for Java test method naming conventions e.g. the test method sortList() will test the sortList() method for all scenarios.